index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Web Design course for everybody website" />
+    <meta name="keywords" content="Web,Web Design, course, HTML, CSS, Javascript, ArthurGC, CodeClub" />
+    <meta name="author" content="Arthur Guzman" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/public/favicon-32x32.svg" />
+    <script
+      src="https://kit.fontawesome.com/27e53b408d.js"
+      crossorigin="anonymous">
+    </script>
+    <link rel="stylesheet" href="css/home/home.css" />
+    <title>Web Design for Everybody: Basics of Web Development and Coding | CodeClub</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="index.html" class="link"><img src="public/logo.svg" alt="code club icon"></a>
+            <ul>
+                <li>
+                    <a href="about.html" class="link">About</a>
+                </li>
+                <li>
+                    <a href="#" class="link">Program</a>
+                </li>
+                <li>
+                    <a href="#" class="link">Join for Free</a>
+                </li>
+            </ul>
+            <i class="fas fa-bars hamb"></i>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <p class="headline__greet">"Hello World!"</p>
+            <h1 class="headline__title">Web Design for Everybody: Basics of Web Development and Coding</h1>
+            <div class="headline__dsc">
+                <p>Learn to Design and Create Websites. Build a responsive and accessible web portfolio using HTML5, CSS3, and JavaScript.</p>
+            </div>
+            <p class="headline__date">2021.07.21(TUE) ~ 22(THU)</p>
+        </section>
+        <section class="program">
```

Comment:
This section should look like the screenshot below on the desktop
![right5](https://user-images.githubusercontent.com/58376020/123713847-93f4cf80-d875-11eb-910e-437ecfffeead.png)